### PR TITLE
The first Quartermaster Milestone Report! Yay.

### PR DESCRIPTION
**Pull request by mirkoboehm**
Needs review, though. Feedback welcome.

### CODE REVIEW

File: content/blog/qmstr-milestone-1-report.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+the end of every sprint, a public video conference will be held where
+the sprint results are demonstrated, and the next sprint is planned in
+detail.
+
+### Open Governance
+
+Everybody interested in Quartermaster is invited to join the public
+sprint reviews and milestone requirements workshops (there is however
+limited capacity for the workshops, so make sure to register early!)
+The project applies an open, meritocratic governance model: Our code
```
Comment (lisa-noeth) [inline content/blog/qmstr-milestone-1-report.md]: our code

File: content/blog/qmstr-milestone-1-report.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+and the roadmap including tickets are on Github, and there is an open
+Slack workspace that everybody is invited to join. In general, all
+aspects of the project are open for collaboration, it is truly Open
+Source.
+
+### Have some standards, please...
+
+There are some norms though that the project applies: Everything
```
Comment (lisa-noeth) [inline content/blog/qmstr-milestone-1-report.md]: : everything

File: content/blog/qmstr-milestone-1-report.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+to go hang out and party with the other 2 year olds. If the theme of
+version 0.1 was "the workflow toolchain", the theme of milestone two
+will be "documentation". On April 11, 2018 we held the milestone 2
+requirements workshop. During the Legal and Licensing Workshop 2018,
+participants gave feedback and identified more requirements. This
+input was poured into a milestone roadmap roughly represented by the
+tickets in
+the
+[QMSTR super repository](https://github.com/QMSTR/qmstr-all/milestones?direction=asc&sort=due_date&state=open). After
+some housekeeping, we will implement implement SPDX metadata handling
```
Comment (lisa-noeth) [inline content/blog/qmstr-milestone-1-report.md]: implement implement (2x)

**Review by lisa-noeth — COMMENTED** [review]
Just a few spelling errors. Capitals after colons, one doubled word. 

Other than that it looks great!

File: content/blog/qmstr-milestone-1-report.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+for analysis, and extend the current reporting modules. Some effort
+will be invested into creating an automated integration testing
+environment to ensure the stability of the toolchain. Details for the
+final three sprints will be identified as we go (or maybe based on
+your feedback?) Expect version 0.2 in early July, and the next
```
Comment (Katajau) [inline content/blog/qmstr-milestone-1-report.md]: after the brackets full stop

**Comment by Katajau** [comment]
Looks good :-)

**Comment by GiasemiSh** [comment]
lgtm

**Review by GiasemiSh — APPROVED** [review]

**Pull request merged**